documentation/jetty-documentation/src/main/asciidoc/old_docs/security/authentication.adoc

```diff
@@ -24,7 +24,7 @@ Authorization::
 
 ==== Configuring an Authentication mechanism
 
-Jetty server supports several standard authentication mechanisms: http://en.wikipedia.org/wiki/Basic_access_authentication[BASIC]; http://en.wikipedia.org/wiki/Digest_authentication[DIGEST]; http://en.wikipedia.org/wiki/Form-based_authentication[FORM]; CLIENT-CERT; and other mechanisms can be plugged in using the extensible http://docs.oracle.com/cd/E19462-01/819-6717/gcszc/index.html[JASPI] or http://en.wikipedia.org/wiki/SPNEGO[SPNEGO] mechanisms.
+Jetty server supports several standard authentication mechanisms: http://en.wikipedia.org/wiki/Basic_access_authentication[BASIC]; http://en.wikipedia.org/wiki/Digest_authentication[DIGEST]; http://en.wikipedia.org/wiki/Form-based_authentication[FORM]; CLIENT-CERT; and other mechanisms can be plugged in using the extensible https://jakarta.ee/specifications/authentication/2.0/jakarta-authentication-spec-2.0.pdf[JakartaAuthentication] or http://en.wikipedia.org/wiki/SPNEGO[SPNEGO] mechanisms.
```

Comment:
It's not the jakarta specs here, that's only jetty-11.

Reply:
Forgot to change this. I have now reverted back to the original link.